### DISCUSSION TITLE
[Feature] Helpertype = player QOL improvements

### DISCUSSION THREAD

**Discussion by Amidweiz** [ikemen-engine/Ikemen-GO#884] [Feature Request] [answered]
Since my [https://github.com/K4thos/Ikemen_GO/wiki/wip#ikemenversion](IkemenVersion) enhancement is being taken into note. Here's some requests from the mugen community with some enhancements to `helpertype = player`. Though these could also be applied to a normal helper if deemed beneficial for both.

- Being able to access and be affected by the negative states (-2, -3, -4, etc)
- Being able to track the opponent like a normal player if jumped over, in turn ignoring the assigned `facing =` trigger
- Being able to define the `[Data]` of a player helper
- Helpers that hit each other shouldn't increment the hit counter

**Comment by Windblade-GR01** [comment, marked as answer]
> Being able to access and be affected by the negative states (-2, -3, -4, etc)

Already posible on both types of helpers

> Being able to track the opponent like a normal player if jumped over, in turn ignoring the assigned facing = trigger 

Already posible too.

> Helpers that hit each other shouldn't increment the hit counter

Understood. This is not currently posible. We will take it into account.

> Helpers that hit each other shouldn't increment the hit counter

This can be added but it will not be enabled by default.
Because helper based superarmor exists.
